README.md

```diff
@@ -1278,7 +1313,7 @@ client.mutateGraph(query).enqueue(new GraphCall.Callback<Storefront.Mutation>()
 
 #### Polling for checkout completion [⤴](#table-of-contents)
 
-After a successful `checkoutCompleteWith...` mutation, the checkout process has started. This process is usually short but not immediate so polling is required to obtain an updated checkout in a `ready` state - with a `Storefront.Order`.
+After a successful `checkoutCompleteWith...` mutation, the checkout process starts. This process is usually short, but it isn't immediate. Because of this, polling is required to obtain an updated checkout in a `ready` state - with a `Storefront.Order`.
```

Comment:
does `Storefront.Order` actually return `ready` as a value (instead of `null`) when the checkout is complete?